### ISSUE TITLE
will it be made into a package on its own?

### ISSUE THREAD

**Issue by Boxerbuzz** [renancaraujo/letspicture#1]
Am just wondering if it will be made into a package of it own, i would love to give it at test run

**Comment by Boxerbuzz** [comment]
PS: its not really an issue, am just wondering.

**Comment by renancaraujo** [comment]
Yep: [niks](https://github.com/renancaraujo/niks) and [bitmap](https://github.com/renancaraujo/bitmap). I'm preparing readmes and tutorials. Stay tuned.

**Comment by leynier** [comment]
Hi @renancaraujo, do you still plan to do those tutorials?

If you already made them, could you tell us where to find those tutorials?

The application is very good and efficient, but I find that it uses an old version of the `bitmap` and `rxdart` packages, I am trying to use its `niks` and` bitmap` packages but without `sqflite` and` mobx`.

Thank you very much in advance.

**Comment by renancaraujo** [comment]
I got to put this project on hold due to time management, but this is somethign that im looking forward to do in the future.
